src/app/loginComponents/accounts/external/accounts.component.ts

```diff
@@ -238,4 +248,43 @@ export class AccountsExternalComponent implements OnInit, OnDestroy {
     this.ngUnsubscribe.next();
     this.ngUnsubscribe.complete();
   }
+
+  revokeToken() {
+    let dialogRef = this.dialog.open(RevokeTokenDialogComponent, {
+      width: bootstrap4largeModalSize,
+    });
+    dialogRef.afterClosed().subscribe((result) => {
+      if (result) {
+        this.deleteToken(TokenSource.DOCKSTORE).subscribe(
```

Comment:
Thanks for the hard work on this!  Before you make the `deleteToken` request, the docs for `AlertService` recommend that you include an `alertService.start` that describes what is being done.  See here for more info:
https://github.com/dockstore/dockstore-ui2/blob/08fca140d1f5681ef8b6d46de282192701261ef4/src/app/shared/alert/state/alert.service.ts#L23

Reply:
Added - thanks for letting me know!